src/pages/resultsView/enrichments/EnrichmentsUtil.tsx

```diff
@@ -550,6 +762,22 @@ export function pickMethylationEnrichmentProfiles(
     });
 }
 
+export function pickAllGenericAssayEnrichmentProfiles(
+    profiles: MolecularProfile[]
+) {
+    // TODO: Pick profiles from all Generic Assay dataTypes after we implement related features
+    return profiles.filter(p => {
+        return (
+            p.molecularAlterationType ===
+                AlterationTypeConstants.GENERIC_ASSAY &&
+            (((p.datatype === DataTypeConstants.BINARY ||
+                p.datatype === DataTypeConstants.CATEGORICAL) &&
+                p.patientLevel === false) ||
```

Comment:
I remember the backend is supporting the patient level while I was doing the backend review. But we can leave this as is and enable it once we confirmed that this feature can support patient-level data.

Reply:
Could you add a comment, e.g. `// TODO: enable patient-level profile after confirming patient-level data is compatible with enrichment feature`

Reply:
These conditions are saying not to display patient level data in frontend. Backend may return something but now like you mentioned there maybe some display problem. Sure I will add some comments here.